tests/unit/testResources/schema.ts

```diff
@@ -10,6 +10,7 @@ export const TEST_CCLOUD_SCHEMA = Schema.create({
   environmentId: TEST_CCLOUD_SCHEMA_REGISTRY.environmentId,
   connectionId: TEST_CCLOUD_SCHEMA_REGISTRY.connectionId,
   connectionType: TEST_CCLOUD_SCHEMA_REGISTRY.connectionType,
+  isHighestVersion: true,
```

Comment:
linting over the tests caught that these test-suite instantiation of Schema objects was lacking a param. Makes me wonder how this wasn't caught before.